chirp/drivers/ft1d.py

```diff
@@ -1045,10 +1046,7 @@ def get_memory(self, number):
             mem.offset = int(_mem.offset) * 1000
             mem.rtone = mem.ctone = chirp_common.TONES[_mem.tone]
             self._get_tmode(mem, _mem)
-            if mem.duplex is None:
-                mem.duplex = DUPLEX[""]
-            else:
-                mem.duplex = DUPLEX[_mem.duplex]
+            mem.duplex = DUPLEX[_mem.duplex] if _mem.duplex else ""
```

Comment:
The old code looks definitely wrong, but ... I'm not sure `_mem.duplex` would ever equate to `False` since it's a bitwise object. Does this actually fix a problem or are you just "gardening" ?